mobile/native/src/event/api.rs

```diff
@@ -34,6 +34,7 @@ pub enum Event {
 pub enum BackgroundTask {
     AsyncTrade(OrderReason),
     Rollover(TaskStatus),
+    RecoverDlc(TaskStatus),
```

Comment:
🙃 this is a great spot to write some commets (or in the classes). 
e.g. I was away at a conference, and some of the names are pretty cryptic. 
more docs would be great!